src/transform/FooterContainer.css

```diff
@@ -1,43 +1,27 @@
-.pagelib_footer_container {
-  background-color: #eaecf0;
+.pagelib_footer_section {
+  padding-bottom: 1.5em;
 }
 
-.pagelib_theme_dark .pagelib_footer_container {
-  background-color: #222;
-}
-
-.pagelib_theme_sepia .pagelib_footer_container {
-  background-color: #e1dad1;
-}
-
-.pagelib_theme_black .pagelib_footer_container {
-  background-color: #101418;
+.pagelib_footer_section a:hover {
+  text-decoration: none;
 }
 
 .pagelib_footer_container_heading {
-  padding-top: 35px;
-  font-size: 0.8em;
-  line-height: 0.8em;
-  color: #72777D;
 }
 
 .pagelib_theme_sepia .pagelib_footer_container_heading {
```

Comment:
Any reason to keep those rules empty? Why not delete it?

Reply:
Not sure. There had been several empty rules in this CSS file before. @joewalsh Any ideas?

Reply:
I think all these empty rules can be removed. Should remove the classes in the HTML as well. They might need to be re-added depending on design tweaks.

Reply:
Ok, I'll do that

Reply:
Done